openrtb_ext/imp_appnexus_Keywords_test.go

```diff
@@ -7,29 +7,27 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestKeywordsUtilUnmarshalJSON(t *testing.T) {
+func TestKeywordsUnmarshalJSON(t *testing.T) {
 	type keywords struct {
 		Keywords Keywords `json:"keywords"`
 	}
 
-	t.Run("dynamic-json", func(t *testing.T) {
-		jsonData := []byte(`{"keywords" : { "pets": ["dog"] }}`)
-		var keywords keywords
-		assert.NoError(t, json.Unmarshal(jsonData, &keywords))
-		assert.Equal(t, "pets=dog", keywords.Keywords.String())
-	})
+	type validTest struct {
```

Comment:
Consider renaming this struct to `testCase` as this struct defines a test case for the keywords unmarshal. It'd make sense to name this `validTest` if you had say two different structs for valid and invalid test cases but in this case we're just gonna use this one struct for both valid and invalid test cases.

Reply:
As part of [825b164](https://github.com/prebid/prebid-server/pull/2649/commits/825b1640408890f3c12a4196a6114b72fb15b1e6), I have added invalid test cases. So keeping the names as `validTestCases` and `invalidTestCases`